kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -5,11 +5,22 @@ The sample scripts demonstrate the creation of a WebLogic domain home in a Docke
 ## Prerequisites
 
 The following prerequisites must be handled prior to running the create domain script:
-* The WDT sample requires that JAVA_HOME is set to a java JDK version 1.8 or greater
-* Make sure the WebLogic operator is running.
-* The operator requires WebLogic Server 12.2.1.3.0 with patch 28076014 applied. Refer to [Weblogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how to create one. If a different `domainHomeImageBase` (see Configuration table below) is specified, the specified image needs to be built locally or pulled from a repository.
+* The WDT sample requires that `JAVA_HOME` is set to a Java JDK version 1.8 or later.
+* Make sure that the WebLogic operator is running.
+* The operator requires WebLogic Server 12.2.1.3.0 with patch 28076014 applied. Refer to [WebLogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how to create one. If a different `domainHomeImageBase` (see Configuration table below) is specified, the specified image needs to be built locally or pulled from a repository.
```

Comment:
The second sentence can be removed. It is more distracting than helping, I think.